frameworks/mxnet/get_started_mnist_deploy.ipynb

```diff
@@ -0,0 +1,608 @@
+    "    \n",
+    "    bucket = CONFIG['public_bucket']\n",
+    "    s3 = boto3.client('s3')\n",
+    "    key = 'datasets/image/MNIST/model/mxnet-training-2020-11-21-01-38-01-009/model.tar.gz'\n",
+    "    target = os.path.join('/tmp', 'model.tar.gz')\n",
+    "\n",
+    "    s3.download_file(bucket, key, target)\n",
+    "    \n",
+    "    # upload to default bucket\n",
+    "    mx_mnist_model_data = sess.upload_data(\n",
+    "        path=os.path.join('/tmp','model.tar.gz'),\n",
+    "        bucket=sess.default_bucket(),\n",
+    "        key_prefix='model/mxnet')\n",
+    "\n",
+    "    \n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(mx_mnist_model_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## MXNet Model Object\n",
+    "\n",
+    "The `MXNetModel` class allows you to define an environment for making inference using your\n",
+    "model artifact. Like `MXNet` class we discussed \n",
+    "[in this notebook for training an MXNet model](\n",
+    "get_started_mnist_train.ipynb), it is high level API used to set up a docker image for your model hosting service.\n",
+    "\n",
+    "Once it is properly configured, it can be used to create a SageMaker\n",
+    "Endpoint on an EC2 instance. The SageMaker endpoint is a containerized environment that uses your trained model \n",
+    "to make inference on incoming data via RESTful API calls. \n",
+    "\n",
+    "Some common parameters used to initiate the `MXNetModel` class are:\n",
+    "- entry_point: A user defined python file to be used by the inference image as handlers of incoming requests\n",
+    "- source_dir: The directory of the `entry_point`\n",
+    "- role: An IAM role to make AWS service requests\n",
+    "- model_data: the S3 bucket URI of the compressed model artifact. It can be a path to a local file if the endpoint \n",
+    "is to be deployed on the SageMaker instance you are using to run this notebook (local mode)\n",
+    "- framework_version: version of the MXNet package to be used\n",
+    "- py_version: python version to be used\n",
+    "\n",
+    "We elaborate on the `entry_point` below.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = MXNetModel(\n",
+    "    entry_point='inference.py',\n",
+    "    source_dir='code',\n",
+    "    role=role,\n",
+    "    model_data=mx_mnist_model_data,\n",
+    "    framework_version='1.7.0',\n",
+    "    py_version='py3'\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "os.system(\"docker container ls | grep 8080 | awk '{print $1}' | xargs docker container rm -f\")\n",
+    "\n",
+    "model = MXNetModel(\n",
```

Comment:
here's the second

Reply:
Good catch. That cell is used for debugging. Forgot to delete.